scratch/copy-mk.py

```diff
@@ -126,9 +127,7 @@ async def async_main(args) -> int:
         "band": band,
     }
 
-    out_cmd = [
-        "./sim_correlator.py",
-    ]
+    out_cmd = [str(os.path.join(os.path.dirname(__file__), "sim_correlator.py"))]
```

Comment:
Why `str(...)`? os.path.join already returns a `str`.

Reply:
I ... don't remember why I did this.
I had some kind of trouble where it complained about types and it went away when I added `str` here but there may have been another issue elsewhere that has since been fixed.

Will remove.